flake.nix

```
@@ -222,15 +222,16 @@
             } // extraArgs);
         in
         rec {
-          nickel = buildPackage { pname = "nickel-lang"; };
+          nickel-lang = buildPackage { pname = "nickel-lang"; };
+          nickel = buildPackage { pname = "nickel-lang-cli"; };
```

Just to avoid confusion

```suggestion
          nickel-lang-cli = buildPackage { pname = "nickel-lang-cli"; };
```